### PR TITLE
Task 369-372: multi_woz_v22

### PR DESCRIPTION
**Pull request by ayushkalani**
All checks passed at 23:56, 14th September

### CODE REVIEW

**Comment by swarooprm** [comment]
@Palipoor, wondering if you can help review this PR?

**Review by Palipoor — COMMENTED** [review]
Hi! 
1. In task 369 the sentence "Your job is to predict the appropriate sentence which is coherent with the given sentences." is redundant. 
2. In task 370, the outputs of instances are not correctly formed. Each element of the list should be one accepted output, so for example, this:
```
            "output": [
                "5",
                "4",
                "3",
                "2",
                "10",
                "9",
                "8",
                "1",
                "6",
                "7"
            ]
```
Should be:
```
"output" : [  "[5,4,3,2,10,9,8,1,6,7]"  ]
```

3. In task 371 you don't need to include "Answer: " in every output.
4. I'm not sure about the category "User Utterance Generation". It may be fit into one of the other categories, maybe just "Text Generation".
5. In task 372, maybe change "based on that you have to identify what will be user's command for that particular reply"  to "based on that you have to identify what  user's command for that particular reply have been"
6. In task 372, are all inputs from "system"s replies? Because it seems like some of them are from the users. If so, please include that in the definition.

File: tasks/README.md [hunk 1]
```diff
@@ -6,4 +6,8 @@ This directory contains the tasks that are part of this benchmark.
 Name | Summary | Category
 ---- | ----------- | --------
 `task1001_squad1.1_question_generation` | Generating guestions (based on SQuAD 1.1) | Question Generation  
-`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
\ No newline at end of file
+`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
+`task369_multi_woz_sentence_generation` | Generating next sentences based on previous and future sentences | Sentence Generation
```
[the remark this answers is not in the document]
Comment (ayushkalani) [reply tasks/README.md]: Done.

File: tasks/README.md [hunk 2]
```diff
@@ -6,4 +6,8 @@ This directory contains the tasks that are part of this benchmark.
...
+`task370_multi_woz_ordering_answer_generation` | Identifying the correct order of a conversation flow | Answer Generation
+`task371_multi_woz_classification` | Classifying dialouse into User and System | Classification
```
[the remark this answers is not in the document]
Comment (ayushkalani) [reply tasks/README.md]: Typo, my bad.

**Comment by ayushkalani** [comment]
@Palipoor Addressed all review comments. Thanks for your feedback.

6. Task 372. I have included both user and system replies because extracting a single entity reply was getting intermittent issues. So I have changed the definition for that task.

cc: @swarooprm

**Comment by Palipoor** [comment]
Thanks! They look good to me.

**Comment by swarooprm** [comment]
Your HW2 is complete now.

Would you like to submit the same tasks to our big project repo https://github.com/allenai/natural-instructions-expansion/pulls? You just need to fork that repo, rename tasks starting from the latest number in the repo and create PR there. This way, you can be part of our research on "learning from instructions instead of learning from examples (conventional machine learning)". This is purely optional and your HW2 grade does not depend on this @ayushkalani

**Pull request closed without merging**